recipe/meta.yaml

```
@@ -36,12 +45,12 @@ requirements:
 
 test:
   imports:
-    - xrayutilities
-    - xrayutilities.analysis
-    - xrayutilities.io
-    - xrayutilities.materials
-    - xrayutilities.math
-    - xrayutilities.simpack
+    - xrayutilities             # [build_platform == target_platform]
+    - xrayutilities.analysis    # [build_platform == target_platform]
+    - xrayutilities.io          # [build_platform == target_platform]
+    - xrayutilities.materials   # [build_platform == target_platform]
+    - xrayutilities.math        # [build_platform == target_platform]
+    - xrayutilities.simpack     # [build_platform == target_platform]
```

Keep this as-is.`test: native_and_emulated` is taking care that these are only run if they could be successful.
```suggestion
    - xrayutilities
    - xrayutilities.analysis
    - xrayutilities.io
    - xrayutilities.materials
    - xrayutilities.math
    - xrayutilities.simpack
```